clients/src/main/resources/common/message/BeginQuorumEpochRequest.json

```diff
@@ -18,24 +18,37 @@
   "type": "request",
   "listeners": ["controller"],
   "name": "BeginQuorumEpochRequest",
-  "validVersions": "0",
-  "flexibleVersions": "none",
+  "validVersions": "0-1",
```

Comment:
Please add a comment to mention what changed in v.1.

Reply:
Done. I just realized that I missed one part of the request handling of this RPC.